src/dotnet/commands/dotnet-new/Program.cs

```diff
@@ -33,7 +33,7 @@ public int CreateEmptyProject(string languageName, string templateDir, bool isMs
             // Check if project.json exists in the folder
             if (File.Exists(Path.Combine(Directory.GetCurrentDirectory(), "project.json")) && !isMsBuild)
             {
-                Reporter.Error.WriteLine($"Creating new {languageName} project failed, project already exists.");
+                Reporter.Error.WriteLine($"{LocalizableStrings.CreatingNewError}{languageName}{LocalizableStrings.CreatingNewError2}");
```

Comment:
In this case, I prefer something along the lines of string.Format(LocalizableStrings.CreatingNewProjectAlreadyExists, languageName); where CreatingNewProjectAlreadyExists holds the whole sentence. I believe this is cleaner and also the sentences to be translated will be complete for translators all in the LocalizableStrings. Otherwise, people would have to scan the code to get the full context of the sentences.